tests/heating_function_depth.prm

```diff
@@ -0,0 +1,68 @@
+# A test for the function heating plugin.
+# The heating function is given in terms of depth.
```

Comment:
If these tests are all the same, with the only difference being the coordinate system, can you include the one that uses the cartesian coordinates and only keep the changes in this file?